tests/src/test/java/alluxio/worker/AlluxioWorkerRestApiTest.java

```diff
@@ -91,6 +97,37 @@ public void getCapacityBytesTest() throws Exception {
         NO_PARAMS, HttpMethod.GET, capacityBytes).run();
   }
 
+  private Configuration mockConfiguration() {
```

Comment:
Could we avoid using powermock in this class by calling `WorkerContext.getConf().set(key, value)` directly, then resetting the worker context after the test is complete? The resetting might require us to add a util method which uses whitebox to call `WorkerContext.reset()`